cmd/argocd/commands/admin/settings_rbac.go

```diff
@@ -72,31 +72,34 @@ var validRBACActions map[string]bool = map[string]bool{
 
 // NewRBACCommand is the command for 'rbac'
 func NewRBACCommand() *cobra.Command {
+	var (
+		clientConfig clientcmd.ClientConfig
+	)
 	var command = &cobra.Command{
 		Use:   "rbac",
 		Short: "Validate and test RBAC configuration",
 		Run: func(c *cobra.Command, args []string) {
 			c.HelpFunc()(c, args)
 		},
 	}
-	command.AddCommand(NewRBACCanCommand())
-	command.AddCommand(NewRBACValidateCommand())
+	clientConfig = cli.AddKubectlFlagsToCmd(command)
+	command.AddCommand(NewRBACCanCommand(clientConfig))
+	command.AddCommand(NewRBACValidateCommand(clientConfig))
 	return command
 }
 
 // NewRBACCanRoleCommand is the command for 'rbac can-role'
-func NewRBACCanCommand() *cobra.Command {
+func NewRBACCanCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
```

Comment:
Any reason to put the clientConfig as a parameter? If we are taking namespace as an input, do we need to fetch the namespace from clientConfig rather than adding namespace as a variable like `policyFile`?

Reply:
I've modified this back, at the time I missed something (most likely the issue you commented below 
`the flag &policyFile is repeated. This should be &namespace?` and while testing the command it repeatedly returned some segmentation fault so I thought it was due to this and putting the clientConfig in params fixed it.

I've changed it back now that I know why my command was failing at the time